index.html

```diff
@@ -227,7 +227,7 @@
   }
   function renderStudent(studentObj) {
     var name = studentObj.firstname + " " + studentObj.lastname;
-    var email = studentObj.firstname[0] + studentObj.lastname + "@eastsideprep.org";
+    var email = studentObj.email;
```

Comment:
There are other places where we are generating emails on the client, e.g. in the class expando view